mrblib/loops/master.rb

```diff
@@ -4,10 +4,12 @@
   if button_a_pressed?
     puts "Button A pressed"
     puts_color(0, 30, 160, 100, "RED");
+    puts_screen(60, 60, "BUTTON A");
```

Comment:
Major method "puts" is a method for output strings.
So, puts_color is a little bit strange name for me.

```
e.g. 
puts_color -> put_color 
puts_screen -> put_strings
```

Reply:
@shuheiokuda0704  I will fix accordingly